calico-cloud/network-policy/recommendations/learn-about-policy-recommendations.mdx

```diff
@@ -0,0 +1,145 @@
+---
+description: Learn more about policy recommendations.
+---
+
+# Learn more about policy recommendations
+
+## Big picture
+
+To use the [policy recommendation feature](policy-recommendations.mdx), you must have traffic between endpoints. In this tutorial we show you how recommendations are generated using flow logs in your cluster, for traffic to/from namespaces, network sets, private network IPs and public domains.
+
+### Configure felix for fast flow logs collection
```

Comment:
### Create resources for the tutorial

Because the policy recommendation feature requires traffic between endpoints, this step provides a  YAML to create these resources.  If your cluster is already generating traffic for policy recommendations, you can skip this step and follow along with your own cluster.
 
Add a suggestion, <Cmd+g>
Add heading textAdd bold text, <Cmd+b>Add italic text, <Cmd+i>
Add a quote, <Cmd+Shift+.>Add code, <Cmd+e>Add a link, <Cmd+k>
Add a bulleted list, <Cmd+Shift+8>Add a numbered list, <Cmd+Shift+7>Add a task list, <Cmd+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply
Slash commands

Leave a comment
No file chosen
Attach files by dragging & dropping, selecting or pasting them.
Styling with Markdown is supported

```bash
kubectl patch felixconfiguration.p default -p '{"spec":{"flowLogsFlushInterval":"10s"}}'
```

### Apply the tutorial YAML, creating the necessary resources

```bash
kubectl create -f {{tutorialFilesURL}}/00-namespace.yaml
```
```suggestion
### Configure felix for fast flow logs collection
```